src/components/action-chip/ActionChip.vue

```diff
@@ -34,8 +34,7 @@
 </template>
 
 <script lang="ts">
-import Vue from 'vue'
-import { Component, Prop } from 'vue-property-decorator'
+import { Component, Prop, Vue } from 'vue-facing-decorator'
```

Comment:
For Vue3,  `vue-property-decorator` is replaced with `vue-facing-decorator`. This change is done in all the components to avoid all the console errors.

Reply:
I support this. Thanks!